docs/examples/site_config.yml

```diff
@@ -0,0 +1,225 @@
+---
+plugins:
+  marathon:
+    fields:
+      apps:
+        # Build 'id' field from 'group' and 'name' fields defined below
+        id:
+          default: '{{ APP.group }}/{{ APP.name }}'
+        # Add custom 'name' field
+        name:
+          required: true
+        # Add custom 'group' field with default of SERVICE_NAME var
+        group:
+          required: true
+          default: '{{ VARS.SERVICE_NAME }}'
+        image:
+          # Use DOCKER_IMAGE var if available, otherwise use "group" and known prefix
+          # Note: the hyphens in the jinja tags strip newlines within the template, and
+          # the |- strips off the trailing newline in the YAML block
+          default: |-
+            {%- if VARS.DOCKER_IMAGE is defined -%}
+            {{ VARS.DOCKER_IMAGE }}
+            {%- else -%}
+            1234567890.dkr.ecr.us-east-1.amazonaws.com/{{ APP.group }}{{ ( ':' ~ VARS.IMAGE_TAG ) if VARS.IMAGE_TAG is defined else '' }}
+            {%- endif -%}
+        # Set reasonable defaults for 'cpus', 'mem', and 'disk'
+        cpus:
+          default: 0.5
+        mem:
+          default: 512
+        disk:
+          default: 10
+        instances:
+          default: 3
+        # Default constraints make sure service instances are spread out across availability zones
+        constraints:
+          default:
+            - [ "@region", "IS", "aws/us-east-1" ]
+            - [ "@zone", "GROUP_BY", "3" ]
```

Comment:
Comment these, but leave them, mentioning that fault-domain detection is an Enterprise feature